### PR TITLE
Typos found by codespell

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**

### CODE REVIEW

File: man/systemd-sysusers.xml [hunk 1]
```diff
@@ -83,7 +83,7 @@
 
       <varlistentry>
         <term><option>--replace=<replaceable>PATH</replaceable></option></term>
-        <listitem><para>When this option is given, one ore more positional arguments
+        <listitem><para>When this option is given, one or more positional arguments
```
Comment (keszybz) [inline man/systemd-sysusers.xml:86]: But please read this out loud: "one! ore! more!", it's beautiful!

File: src/shared/linux/bpf.h [hunk 2]
```diff
@@ -4625,7 +4625,7 @@ enum {
 	BPF_SOCK_OPS_TIMEOUT_INIT,	/* Should return SYN-RTO value to use or
 					 * -1 if default value should be used
 					 */
-	BPF_SOCK_OPS_RWND_INIT,		/* Should return initial advertized
+	BPF_SOCK_OPS_RWND_INIT,		/* Should return initial advertised
```
Comment (keszybz) [inline src/shared/linux/bpf.h]: Those files are imported verbatim from the linux kernel, please don't do any fixes under `src/shared/linux/`. (Any fixes should be submitted to the kernel upstream.)

File: src/shared/import-util.c [hunk 3]
```diff
@@ -90,7 +90,7 @@ int import_url_change_suffix(
         /* This drops the specified number of path components of the specified URI, i.e. the specified number
          * of non-empty substring between two "/" characters from the end of the string, and then append the
          * specified suffix instead. Before doing all this it chops off the "Query" and "Fragment" suffixes
-         * (they are *not* readded to the final URL). Note that n_drop_components may be 0 (in which case the
+         * (they are *not* read to the final URL). Note that n_drop_components may be 0 (in which case the
```
Comment (keszybz) [inline src/shared/import-util.c]: This was "re-added", please either add the dash or drop this chunk.

Comment (keszybz) [inline src/shared/import-util.c]: This was "re-added", please either add the dash or drop this chunk.

File: src/shared/linux/bpf.h (hunk 2 above)
Comment (keszybz) [reply src/shared/linux/bpf.h]: This chunk is still here.

Comment (DimitriPapadopoulos) [reply src/shared/linux/bpf.h]: Same goes for `src/shared/linux/ethtool.h` then.

File: src/shared/import-util.c (hunk 3 above)
Comment (DimitriPapadopoulos) [reply src/shared/import-util.c]: My wrong. Actually `readded` is correct. Also the `readded->read` typo is in the "_rare_" dictionary of codespell, which means that the typo occurs more often than actual voluntary use of `readded`.

Therefore we could keep `readded`, but I find it confusing - at least I was confused but I'm not a native speaker.

Comment (keszybz) [reply src/shared/import-util.c]: Yeah, "re-added" is better.

File: src/shared/linux/bpf.h (hunk 2 above)
Comment (DimitriPapadopoulos) [reply src/shared/linux/bpf.h]: Both have been fixed.

**Pull request merged**